Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -999,15 +999,15 @@ private void HandleOrderEvent(OrderEvent fill)
 
                     try
                     {
-                        // to be called before updating the Portfolio
-                        NewOrderEvent?.Invoke(this, fill);
-
                         _algorithm.Portfolio.ProcessFill(fill);
                         _algorithm.TradeBuilder.ProcessFill(
                             fill,
                             securityConversionRate,
                             feeInAccountCurrency,
                             multiplier);
+
```

Comment:
I think fundamentally prefer this is the other way around, IMHO the portfolio lags the order events. Putting them before is misleading. In IB it can lag by several minutes sometimes. Not sure the whole premise of this PR is a good idea.